packages/sanity/src/core/studio/packageVersionStatus/PackageVersionStatusProvider.tsx

```diff
@@ -0,0 +1,68 @@
+import {Box, Button, Stack, useToast} from '@sanity/ui'
+import {type ReactNode, useEffect, useState} from 'react'
+import {SANITY_VERSION} from 'sanity'
+import semver from 'semver'
+
+import {hasSanityPackageInImportMap} from '../../environment/hasSanityPackageInImportMap'
+import {useTranslation} from '../../i18n'
+import {checkForLatestVersions} from './checkForLatestVersions'
+
+/*
+ * We are currently only checking to see if the sanity module has a new version available.
+ * We can add more packages to this list (e.g., @sanity/vision) if we want to check for more.
+ */
+const currentPackageVersions: Record<string, string> = {
+  sanity: SANITY_VERSION,
```

Comment:
Just laying out my thoughts, if it makes sense to you as well. 

When we run the bundle we set the `process.env.SANITY_VERSION` reading the package.json so when the bundle is built and uploaded it should have the correct version in the bundle so reading the version should always resolve to the correct version. I am thinking this correctly?

Reply:
That's my understanding! Brief manual testing has show it defined to what I expect, which you can also see in the navbar, for example (it respects tagged versions et al).

This won't be available for other packages (as of now) so we should strategize when that happens.

Reply:
Discussed that this can be addressed later